tools/executor/util.js

```diff
@@ -0,0 +1,60 @@
+const execaRaw = require('execa');
+const AWS = require('aws-sdk');
+
+const AWS_REGION = 'us-east-2';
```

Comment:
What do we do for other regions?

Reply:
Canada!

Reply:
Doh, of course we're running the Canadian PL on Canadian servers 😂 hmm... that region can still pull containers from `us-east-2`, right? I imagine it wouldn't be as fast, but I'd also rather not have to push the image to 3+ registries on every CI run.

Reply:
Talked with Matt about this, we *do* actually want a local copy of this in the Canada region. Should update the the GitHub Actions workflow to push to both registries, but only on `master` builds, as we don't have a Canadian staging environment we need the PR builds to be available for.